README.md

```diff
@@ -0,0 +1,52 @@
+# Moodle
+
+## Installation
```

Comment:
We're all new to Moodle. Please tell what the installation steps are based on … Is it <https://docs.moodle.org/405/en/Installing_Moodle#Command_line_installer> or <https://docs.moodle.org/405/en/Installation_quick_guide> or …?